README.md

```diff
@@ -15,52 +15,62 @@ It is used in conjunction with the `bux-wallet-backend` component, which serves
 
 There are two way of running backend locally:
 
-#### Using ./start.sh script 
+#### Using ./start.sh script
 
-`bux-wallet-frontend` provides a `start.sh` script
-which is using `docker-compose.yml` file to starts up Bux Wallet Frontend, Backend and 
-Bux Server with selected database and cache storage. To start, we need to fill the 
+`spv-wallet-web-frontend` provides a `start.sh` script
+which is using `docker-compose.yml` file to starts up `SPV Wallet` with web-frontend, web-backend and selected database and cache storage.
+
+To start, we need to fill the
 config json which we want to use, there is prepared custom config file `docker/envs/development.json`
-which is used in development environment. This script build local image of bux-wallet-frontend and use 
+which is used in development environment. This script build local image of spv-wallet-web-frontend and use
 already built ones for the rest of applications.
 
 Ports which are used:
-- 3002  - Bux wallet frontend
-- 3003  - Bux server
-- 5432  - PostgreSQL db
-- 6379  - Redis
-- 8080  - Pulse
-- 8081  - Bux wallet backend
+
+- 3002 - SPV wallet web-frontend
+- 8081 - SPV wallet web-backend
+- 3003 - SPV wallet (core service)
+- 5432 - PostgreSQL DB
+- 6379 - Redis
+- 8080 - Block Headers Service
 - 27017 - MongoDB
 
 There are three ways of running this script:
+
 1. With manual configuration - every option is displayed in terminal and user can choose
-   which applications should be started and configure how to run bux-server. Use command:
-  ```bash
-  ./start.sh
-  ```
+   which applications should be started and configure how to run `spv-wallet`. Use command:
+
+```bash
+./start.sh
+```
+
 2. With flags which define how to set up docker services. Ever option is displayed when
    you ran the script with flag `-h` or `--help`.
-  ```bash
-  ./start.sh -db postgresql -c redis -bs true -env development -b false 
-  ```
+
+```bash
+./start.sh -db postgresql -c redis -bs true -env development -b false
+```
+
 3. With `-l/--load` flag. This option add possibility to use previously created `.env.config` file and run whole environment with simple command:
-  ```bash
-  ./start.sh -l
-  ```
+
+```bash
+./start.sh -l
+```
 
 #### Using docker compose
```

Comment:
Actually this shouldn't work anymore 😞 
Probably we should remove this section from README, or adjust `yarn backend` command and this section

Reply:
Which section you talking about? Could you be more specific?

Reply:
Section `#### Using docker compose` sorry, I put a comment on a title of sectio, to be specific, but now I see it is showing more lines.

Reply:
I've removed the whole section.